src/gtfs/write.rs

```diff
@@ -97,7 +98,7 @@ fn ntfs_stop_point_to_gtfs_stop(
         fare_zone_id: sp.fare_zone_id.clone(),
         location_type: StopLocationType::StopPoint,
         parent_station: Some(sp.stop_area_id.clone()),
-        code: None,
+        code: sp.codes.iter().next().map(|c| c.1.clone()),
```

Comment:
@prhod is it correct or I need to get the first `gtfs_stop_code` code?

Reply:
If I read this correctly, you write the first code of the stop, even if it's not a `gtfs_stop_code`. 
This can be a problem for later. 
Can you check the `object_system` value ?
Thanks !